repositories/webhook_events_repository.go

```diff
@@ -43,7 +43,7 @@ func (repo MarbleDbRepository) ListWebhookEvents(ctx context.Context, exec Execu
 		query = query.Where(squirrel.Eq{"delivery_status": mergedFilters.DeliveryStatus})
 	}
 
-	query = query.OrderBy("created_at DESC").Limit(mergedFilters.Limit)
+	query = query.Limit(mergedFilters.Limit)
```

Comment:
for what it's worth, you can add it directly at the end of line 40 (with the query builder, order is not relevant)